src/components/Mailbox.vue

```diff
@@ -488,5 +488,9 @@ export default {
 }
 .empty-content {
 	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: flex-end;
+	margin-bottom: 100px;
```

Comment:
This *might* be problematic with different screen sizes. I suggest testing it with the responsive mode of the browser to see if it also looks good on mobile, tablets and larger screens :)

Reply:
Thank you for reminding me! I completely forgot to test different screen sizes, I unfortunately cannot test them right now as I don't have my laptop with me, but I will test this and add media viewport tags if necessary or change the method I used in general :)

Reply:
I just checked layout for all screen sizes, it works perfectly fine for really small and really large ones :)